examples/sb3_examples/ppo.py

```diff
@@ -115,7 +126,26 @@ def step_wait(self) -> VecEnvStepReturn:
   pass
 
 # Agent trained on envpool version should also perform well on regular Gym env
-test_env = gym.make(env_id)
+if not is_legacy_gym:
+
+  def legacy_wrap(env):
+    env.reset_fn = env.reset
+    env.step_fn = env.step
+
+    def legacy_reset():
+      return env.reset_fn()[0]
+
+    def legacy_step(action):
+      obs, rew, term, trunc, info = env.step_fn(action)
+      return obs, rew, term + trunc, info
+
+    env.reset = legacy_reset
+    env.step = legacy_step
+    return env
+
+  test_env = legacy_wrap(gym.make(env_id))
```

Comment:
btw is it possible to work out-of-box if we don't add `legacy_wrap`?

Reply:
Without the wrap, it wouldn't work using the latest version of sb3 on pip.